### ISSUE TITLE
`circleci orb validate` fails with private dependent orb from same namespace

### ISSUE THREAD

**Issue by jschwanz** [CircleCI-Public/circleci-cli#751]
## Meta:

**CircleCI CLI Version:**
* using image circleci/circleci-cli@sha256:eee9b00ad8929a528171d773e2ffc9d9f82fcd31c2e89a3da9f22bf17bf54bfc
* 0.1.19666+830e7ff (release)

**Operating System:**
Ubuntu 20.04

---

## Current behavior:

I'm attempting to author a private orb that has a dependency on another private orb in the same namespace. When `circleci orb validate` runs as a part of `orb-tools` I get:

```
Error: Cannot find my-namespace/dep-orb@1.2.3 in the orb registry. Check that the namespace, orb name and version are correct.
```

## Expected behavior:

I expect the `orb validate` process to be able to validate the use of private orbs from the same namespace.

**Comment by jschwanz** [comment]
This functionality already exists in `circleci config validate` with the `--org-id` and `--org-slug` arguments. It would be really nice to have it here, too.

**Comment by rafaelqueiroz93** [comment]
Unfortunatelly I'm having the same problem.

Waiting for a solution

**Comment by adamdmharvey** [comment]
We are having this problem with our orbs as well. Effectively, it means we can't really use the `circleci orb validate` command as most of our orbs cross-reference each other now.

**Comment by mcommons1** [comment]
Is there any update on this issue?

**Comment by davidbarkhuizen** [comment]
Wow. CircleCI totally sucks at actually implementing necessary core functionality.  This is just one example of many that I have come across now.  CircleCI - YOU NEED TO RAISE YOUR GAME!!!

**Comment by ADUDevEx** [comment]
After the security breach we experienced in CircleCI we migrated to private orbs and are now blocked by this.

**Comment by kelvintaywl** [comment]
thank you for your sharing folks!

I am tagging our developer experience team here @CircleCI-Public/developer-experience to see if we can offer any updates on this 🙏

**Comment by praveenkeshri** [comment]
?

**Comment by ADUDevEx** [comment]
Thank you @kelvintaywl for your support on this issue. Hopefully the developer-experience team will find time to update this.

**Comment by tdeekens** [comment]
Thanks for the feedback @JulesFaucherre. Has there been any progress on this. Running into this is quite tedious to get around and I am sure fixing it would uplift how companies can adopt and use Orbs more easily.

**Comment by JulesFaucherre** [comment]
Hello, yes sorry for not getting back to this before! This took us a bit more time than expected because we had dependencies on other teams. Now it's a bit more advanced and we should be releasing this fix by the beginning of next week!

**Comment by zbenhadi** [comment]
I've opened a pull request to solve this problem: https://github.com/CircleCI-Public/circleci-cli/pull/922

The `orb validate` will be able to accept org-slug and/or org-id flags just like `config validate`.

**Comment by zbenhadi** [comment]
Hello again ! The PR has been merged and a new cli version has been released fixing this issue (https://github.com/CircleCI-Public/circleci-cli/releases/tag/v0.1.26255)

Thanks to everyone who participated in this issue  !
